src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -1,12 +1,49 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProvider
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceProviderSentReferralSummary
 import java.util.UUID
 
 interface ReferralRepository : JpaRepository<Referral, UUID> {
+  @Query(
+    value = """select sentAt,
+			referenceNumber,
+			assignedToUserName,
+			interventionTitle,
+			serviceUserFirstName,
+			serviceUserLastName from (	
+	select
+			cast(r.sent_at as TIMESTAMP WITH TIME ZONE) as sentAt,
+			r.reference_number as referenceNumber,
+      dfc.id as dynamicFrameworkContractId,
+			au.user_name as assignedToUserName,
+			i.title as interventionTitle,
+			rsud.first_name as serviceUserFirstName,
+			rsud.last_name as serviceUserLastName,
+			row_number() over(partition by r.id order by ra.assigned_at desc) as assigned_at_desc_seq		
+	from referral r
+			 inner join intervention i on i.id = r.intervention_id
+			 inner join referral_service_user_data rsud on rsud.referral_id = r.id
+			 inner join dynamic_framework_contract dfc on dfc.id = i.dynamic_framework_contract_id
+			 left join dynamic_framework_contract_sub_contractor dfcsc on dfcsc.dynamic_framework_contract_id = dfc.id
+			 left join referral_assignments ra on ra.referral_id = r.id
+			 left join auth_user au on au.id = ra.assigned_to_id
+			 left join end_of_service_report eosr on eosr.referral_id = r.id
+	where
+		  r.sent_at is not null
+		  and not (r.concluded_At is not null
+			and r.end_Requested_At is not null
+			and eosr.id is null)
```

Comment:
probably easier to read if the "not" group is on one line (plus capitalisation for native SQL)
```suggestion
		  and not (r.concluded_at is not null and r.end_requested_at is not null and eosr.id is null)
```